packages/features/ee/workflows/components/WorkflowDetailsPage.tsx

```diff
@@ -1,35 +1,26 @@
-import { useRouter, useSearchParams } from "next/navigation";
+import { useSearchParams } from "next/navigation";
```

Comment:
![Image](https://github.com/user-attachments/assets/0855c467-ccf7-43ea-8c56-7cc339031aea)

There is no edit icon in designs.